sentry/src/main/java-templates/io/sentry/environment/Version.java

```diff
@@ -0,0 +1,9 @@
+package io.sentry.environment;
```

Comment:
No license block?

Reply:
Yeah, we don't require those in Sentry projects. One license to rule the project, haha.